content/frameworks/react/getting-started/index.md

```diff
@@ -0,0 +1,97 @@
+---
+page_type: guide
+title: Getting started
+description: |
+  React is an open-source library that makes building UIs easier. This learning path will cover different APIs and tools within the ecosystem that you should consider using to improve the performance and usability of your application.
+author: houssein
+web_lighthouse: N/A
+web_updated_on: N/A # TODO: update
+web_published_on: N/A # TODO: update
+wf_blink_components: N/A
+---
+
+# Getting started
+
+[React](https://reactjs.org/) is an open-source library that makes building UIs easier. This learning path will cover different APIs and tools within the ecosystem that you should consider using to improve the performance and usability of your application.
+
+## Why is this useful?
+
+There is a lot of content that explain how to build fast and reliable applications by using different techiniques or including new APIs but not many that show how this fits within the tooling that comes with a React app. The guides and codelabs in this section of the site instead cover all of this from the perspective of a React app. Only libraries, APIs and features that are specific to the React ecosystem are mentioned.
+
+<div class="aside caution">
+  The tutorials in this learning path do not focus on:
+
+  <ul>
+    <li>How to use React</li>
+    <li>How React works under the hood</li>
+  </ul>
+
+  Although both of these concepts will be touched on when needed, all the guides and codelabs in this section will instead focus on how to build fast and accessible React sites. For this reason, <strong>a basic knowledge of React is required.</strong>
+</div>
+
+To kick things off, this guide will briefly cover [Create React App](https://facebook.github.io/create-react-app/) which is used to build every application in this section. It will then conclude with a brief summary of all the different topics that will be explored in this learning path.
+
+## Create React App
+
+**Create React App** (CRA) is the easiest way to start building a React application with a build system that contains a module bundler (webpack) and transpiler (Babel). It takes care of these configurations by providing a default setup with a number of these core features baked in.
+
+On a command-line shell, you only need to run the following to create a new application+:
```

Comment:
There's a dangling + after application, weirdly enough. Just need to remove it. Must be a copy paste thing.

Reply:
Oops, fixed